git/util.py

```diff
@@ -392,17 +393,8 @@ def _parse_progress_line(self, line):
 
         # find escape characters and cut them away - regex will not work with
         # them as they are non-ascii. As git might expect a tty, it will send them
-        last_valid_index = None
-        for i, c in enumerate(reversed(line)):
-            if ord(c) < 32:
-                # its a slice index
-                last_valid_index = -i - 1
-            # END character was non-ascii
-        # END for each character in line
-        if last_valid_index is not None:
-            line = line[:last_valid_index]
-        # END cut away invalid part
-        line = line.rstrip()
+        line = self.re_ansi_escape.sub('', line)
+        line.strip()
```

Comment:
Thank you! Once `line = line.strip()` is back I can merge it.

Reply:
Sorry, I must have dropped the change during the rebasing. Because the tests only work on `master` I lose track of where the commits are sometimes. 😆